Manager/Exception/RedirectRouteNotUniqueException.php

```diff
@@ -9,7 +9,7 @@
  * with this source code in the file LICENSE.
  */
 
-namespace Sulu\Bundle\RedirectBundle\Manager;
+namespace Sulu\Bundle\RedirectBundle\Manager\Exception;
```

Comment:
/cc @wachterjohannes not sure about the namespace should we move it out of the manager namespace e.g.: `Sulu\Bundle\RedirectBundle\Exception`